disco/cli/upgrade_cost_analysis.py

```diff
@@ -299,18 +299,22 @@ def run_job(job, config, jobs_output_dir, file_log_level):
         global_config["upgrade_simulation_params"]["pydss_controller"] = (
             config.pydss_controllers.pv_controller.dict(),
         )
+    # create a simulation params dictionary
+    rem_list = ["thermal_upgrade_params", "voltage_upgrade_params", "jobs", "enable_pydss_controllers", "pydss_controllers"]
+    simulation_params_config = dict(config)
```

Comment:
I'm surprised this works. I would have used `config.dict()`. Is the result the same?